k0789885/MyGame.py

```diff
@@ -0,0 +1,110 @@
+Class = "Blank"
+choice = "Blank"
+
+print "What type of person are you?"
+print "1. Warrior"
+print "2. Archer"
+print "3. Mage"
+
+
+
+def bar():
+    print "You push the doors open and are greeted to a room of drunkards and maidens."
+    print "There is a man in a dark cloak in the corner and the bartender is at the counter."
+    while choice != 1 and choice != 2 and choice != 3:
```

Comment:
consider:

`choice not in [1,2,3]`

Does one seem easier to understand than the other?